sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -484,6 +484,35 @@ case class GpuInitCap(child: Expression) extends GpuUnaryExpression with Implici
     input.getBase.toTitle
 }
 
+case class GpuStringRepeat(strs: Expression, repeatTimes: Expression)
+    extends GpuBinaryExpression with ImplicitCastInputTypes {
+  override def left: Expression = strs
+
+  override def right: Expression = repeatTimes
+
+  override def dataType: DataType = strs.dataType
+
+  override def inputTypes: Seq[AbstractDataType] =
+    Seq(TypeCollection(StringType, BinaryType), IntegerType)
+
+  def doColumnar(lhs: GpuColumnVector, rhs: GpuColumnVector): ColumnVector =
+    throw new UnsupportedOperationException(s"Cannot columnar evaluate expression: $this")
```

Comment:
are we deferring this for future tickets? Spark supports column for `repeatTimes`

```
scala> df.show

+---+---+
| _1| _2|
+---+---+
|  a|  0|
|  b|  1|
|  c|  2|
+---+---+

scala> df.createTempView("test")

scala> sql("select repeat(_1, _2) from test").show()

+--------------+
|repeat(_1, _2)|
+--------------+
|              |
|             b|
|            cc|
+--------------+
```

Reply:
Wow, I didn't know that thus didn't implement it in cudf. I may need to do that too.